.gitignore

```diff
@@ -1,9 +1,8 @@
 # Specific files #
 ###################
-test.scss
-test.html
-test.css.map
-test.css
+___test.scss
+___test.html
+___test.css
 
```

Comment:
Just to make them more distinguishable, they'll be removed anyway and replaced with proper tests in a `test/` directory.